src/reynir_correct/wrappers.py

```diff
@@ -187,6 +188,8 @@ def val(
 
 def check_errors(**options: Any) -> str:
     """Return a string in the chosen format and correction level using the spelling and grammar checker"""
+    if type(options.get("input", sys.stdin)) == str:
```

Comment:
Better to use `isinstance()` rather than exact type comparison. Also, it would be cleaner to use an intermediate variable:

```
input = options.get("input", sys.stdin)
if isinstance(input, str): options["input"] = [input]
```

Reply:
Done

Reply:
Had to revert; got the following error:
`self = <_pytest.capture.DontReadFromInput object at 0x7fad7751d400>, args = ()

    def read(self, *args):
>       raise OSError(
            "pytest: reading from stdin while output is captured!  Consider using `-s`."
        )
E       OSError: pytest: reading from stdin while output is captured!  Consider using `-s`.

../p39/lib/python3.9/site-packages/_pytest/capture.py:217: OSError
======================================= short test summary info =======================================
FAILED test/test_allkinds.py::test_grammar - OSError: pytest: reading from stdin while output is cap...
`
Seems to be connected to https://github.com/pytest-dev/pytest/issues/7284 which says it's an error in pytest. It works as-is, and also if the default get value is an empty string or something in that vein.